compat/src/index.d.ts

```diff
@@ -57,13 +58,13 @@ declare namespace React {
 	export function isValidElement(element: any): boolean;
 	export function findDOMNode(component: preact.Component): Element | null;
 
-	export class PureComponent<P = {}, S = {}> extends preact.Component {
+	export abstract class PureComponent<P = {}, S = {}> extends preact.Component {
```

Comment:
I agree with `PureComponent` being abstract. Classes are derived from it so abstract makes sense.

Reply:
It shouldn't be abstract. Otherwise it couldn't be used as `<Suspense>` tag. Imho this is proper declaration:
```
import { Component, RenderableProps, ComponentChild } from '../../src';
...
export class Suspense extends Component<SuspenseProps> {
    render(props?: RenderableProps<SuspenseProps>, state?: Readonly<{}>, context?: any): ComponentChild;
}
```

More reasoning in https://github.com/preactjs/preact/pull/1785

Reply:
My comment here was for `PureComponent`, which should be abstract.

Reply:
Sorry, entered comment into wrong box.